handbook/engineering/code-intelligence/goals.md

```diff
@@ -0,0 +1,45 @@
+# Cloud goals and priorities
+
+## Goals
+
+Track our progress in our [Looker dashboard](https://sourcegraph.looker.com/dashboards/131).
+
+### The majority of code intel results are precise (for languages we support fully)
+
+_Languages we fully support currently include Go, TypeScript, C/C++, and Java._ Languages will be added to this list as we make progress toward our second goal, listed below.
+
+It is imperative that we raise adoption *deeply* within a particular language and ecosystem. The goal of the code intelligence team is to provide fast and precise answers about a user's _universe of code_. This can only happen when that entire universe is indexed. We intend to provide an index for _all_ such universes of code by making progress on two fronts. First, we need to raise excitement for the opportunities adopting precise code intelligence makes possible. Once a user sees what is possible, they should have little reason not to put in the effort of adoption. Second, we need to reduce the friction felt during indexing (especially during adoption). Users tend to perceive only the work we ask them to do to set up precise code intelligence and do not perceive the full benefit of precise code intelligence. Reducing the friction of indexing is a critical step in growing support.
+
+**Milestones:**
```

Comment:
Are these in order of operation? We need to accomplish the first before the second? (obviously some things will be done in parallel, but we've prioritized this list)?

Reply:
Not necessarily, these can basically _all_ be done in parallel (for this goal).

Reply:
Even if they can be done in parallel, they should be ordered by importance (as if we couldn't do them in parallel).

Reply:
Agreed they should be ordered by importance. Let's finish agreeing on what the goals/milestones should be first and order them right after that.